src/utils/pskc.cpp

```diff
@@ -41,9 +41,15 @@ namespace Psk {
 
 void Pskc::SetSalt(const uint8_t *aExtPanId, const char *aNetworkName)
 {
-    const char *saltPrefix = "Thread";
-    int         cur        = 0;
-    int         ret        = kPskcStatus_Ok;
+    const char *saltPrefix      = "Thread";
+    int         cur             = 0;
+    int         ret             = kPskcStatus_Ok;
+    int         remainingSpace  = 0;
+    int         networkNameLen  = 0;
+
+    if(strlen(saltPrefix) + OT_EXTENDED_PAN_ID_LENGTH + strlen(aNetworkName) >= sizeof(mSalt)){
+      otbrLogWarning("Network name too long; will be truncated.");
+    }
```

Comment:
To resolve pretty-check: https://github.com/openthread/ot-br-posix/actions/runs/6590257499/job/17915946140?pr=2072
```suggestion
    const char *saltPrefix     = "Thread";
    int         cur            = 0;
    int         ret            = kPskcStatus_Ok;
    int         remainingSpace = 0;
    int         networkNameLen = 0;

    if (strlen(saltPrefix) + OT_EXTENDED_PAN_ID_LENGTH + strlen(aNetworkName) >= sizeof(mSalt))
    {
        otbrLogWarning("Network name too long; will be truncated.");
```